modules/nw-exposing-service.adoc

```diff
@@ -25,67 +25,57 @@ $ oc project myproject
 ----
 
 ifndef::nodeport[]
-. Run the following command to expose the route:
-+
-[source,terminal]
-----
-$ oc expose service <service_name>
-----
-+
-For example:
+. Run the `oc expose service` command to expose the route:
 +
+
 [source,terminal]
 ----
-$ oc expose service mysql-80-rhel7
+$ oc expose service nodejs-ex
 ----
 +
 .Example output
 [source,terminal]
 ----
-route "mysql-80-rhel7" exposed
+route.route.openshift.io/nodejs-ex exposed
 ----
-
-. Use a tool, such as cURL, to make sure you can reach the service using the
-cluster IP address for the service:
++
+. To verify that the service is exposed, you can use a tool, such as cURL, to make sure the service is accessible from outside the cluster.
++
```

Comment:
The two `+` continuations here (line 41 and 43) aren't actually needed.

Same down on line 57.

Reply:
Thanks -- all fixed.